package.json

```diff
@@ -4,11 +4,11 @@
   "description": "A module for Fastify to parse x-www-form-urlencoded bodies",
   "main": "formbody.js",
   "scripts": {
-    "test": "standard && tap --100 test/**/*.test.js && tsd",
-    "test-ci": "tap --100 test/**/*.test.js --coverage-report=html",
-    "lint": "standard | snazzy",
-    "lint-ci": "standard",
     "codecov": "codecov",
+    "lint": "standard | snazzy",
+    "lint:ci": "standard",
+    "test": "standard && tap --100 \"test/**/*.test.js\" && tsd",
+    "test:ci": "tap --100 \"test/**/*.test.js\" --coverage-report=html",
```

Comment:
I don't think it's useful to generate an html report for coverage in CI.

Reply:
Resolved with https://github.com/fastify/fastify-formbody/pull/66/commits/c43623cde0ee656d898f5c166f3720ae483d4378